apps/client/src/features/editors/welcome/composite/WelcomeProjectList.tsx

```diff
@@ -0,0 +1,41 @@
+import { Button } from '@chakra-ui/react';
+
+import { useOrderedProjectList } from '../../../../common/hooks-query/useProjectList';
+
+import style from '../Welcome.module.scss';
+
+interface WelcomeProjectListProps {
+  loadProject: (filename: string) => Promise<void>;
+}
+
+export default function WelcomeProjectList(props: WelcomeProjectListProps) {
+  const { loadProject } = props;
+  const { data } = useOrderedProjectList();
+
+  return (
+    <tbody>
+      {data.reorderedProjectFiles.map((project) => {
+        if (project.filename === data.lastLoadedProject) {
+          return (
+            <tr className={style.current} key={project.filename}>
+              <td>{project.filename}</td>
+              <td>Loaded from last session</td>
+              <td />
```

Comment:
Maybe a continue button that just closes the menu to not confuse people on how to keep the current project